README.md

```diff
@@ -176,6 +176,24 @@ Each row in `data/build_info.csv` consists of the following:
 - `gradle_version`: if built successfully with Gradle, the Gradle version used. Otherwise, it will be `n/a`
 - `use_gradlew`: if built successfully with the custom `gradlew` script provided with the project itself, it will be `1`. Otherwise it will be `n/a`
 
+## Visualization
+
+We provide a simple website that helps visualize the CVEs and provide necessary pointers to details of the CVEs provided in the dataset.
```

Comment:
Also provide quick instructions on how users can add and visualize their own CVEs, and how that works with config.js?

Reply:
Done; at the end of this section